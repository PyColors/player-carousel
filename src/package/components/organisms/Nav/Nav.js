import React from "react";
import HomePage from "../../../pages/HomePage";
import LiveTV from "../../../pages/LiveTVPage";
import CatchUp from "../../../pages/CatchUpPage";

import Tabs from 'muicss/lib/react/tabs';
import Tab from 'muicss/lib/react/tab';

const Nav = () => (
    <Tabs defaultSelectedIndex={0} className='nav'>
        <Tab value="home" label="Home"><HomePage/></Tab>
        <Tab value="live-tv-2" label="Live TV"> <LiveTV/> </Tab>
        <Tab value="catch-up" label="Catch up"><CatchUp/></Tab>
    </Tabs>
);

export default Nav;
