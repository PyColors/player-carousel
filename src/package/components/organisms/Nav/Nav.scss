.nav {

  .mui-tabs__bar {
    margin-bottom: 30px;
    border-bottom: 1px solid #5D5A62;
  }

  .mui-tabs__bar>li>a {
    padding-left: 12px;
    padding-right: 12px;
    font-size: 14px;
    color: #5D5A62;
    cursor: pointer;
    text-transform: inherit;
  }

  .mui-tabs__bar>li.mui--is-active {

    color: white;
    border-bottom: 1px solid white;
  }

  .mui-tabs__bar>li.mui--is-active>a {
    color: white;
  }
}
