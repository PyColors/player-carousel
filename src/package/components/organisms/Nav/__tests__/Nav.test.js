import React from "react";
import Nav from "../Nav";

it("Should renders Nav page without exploding", () => {
    const wrapper = shallow(<Nav>Hello Jest!</Nav>);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper).toHaveLength(1)
});
