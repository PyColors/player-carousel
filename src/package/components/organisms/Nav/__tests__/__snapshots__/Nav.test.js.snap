// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should renders Nav page without exploding 1`] = `
<Tabs
  className="nav"
  defaultSelectedIndex={0}
  initialSelectedIndex={null}
  justified={false}
  onChange={null}
  selectedIndex={null}
>
  <Tab
    label="Home"
    onActive={null}
    value="home"
  >
    <Connect(HomePage) />
  </Tab>
  <Tab
    label="Live TV"
    onActive={null}
    value="live-tv-2"
  >
     
    <LiveTV />
     
  </Tab>
  <Tab
    label="Catch up"
    onActive={null}
    value="catch-up"
  >
    <CatchUp />
  </Tab>
</Tabs>
`;
